src/components/HomePage/Forms/EditMovieForm.tsx

```diff
@@ -1,132 +1,33 @@
 import React, { FC } from 'react';
```

Comment:
By the way, let's consider merging of Edit and Add Moview forms. Looks a bit the same.

Reply:
Totally agree. Merged these forms and modals in commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a